apps/client/src/pages/builder/sidebars/left/sections/picture/section.tsx

```
@@ -22,20 +30,199 @@ import { PictureOptions } from "./options";
+      setPreviewUrl(objectUrl);
 
-      setValue("basics.picture.url", url);
+      setCropModalOpen(true);
+    }
+  };
+
+  const onCropComplete = async () => {
+    if (!imgRef.current || !selectedFile || !crop) {
+      <Toast variant="error">
+        Unable to process the image. Required elements are missing.
+      </Toast>
```

ESLint

[error] String not marked for translation. Wrap it with <Trans>.

(lingui/no-unlocalized-strings)